src/components/TodosList.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TodoItem from './TodoItem';
+
+function TodosList({
+  todosProps, handleChange, delTodo, setUpdate,
+}) {
```

Comment:
-[ ] Optional: It is fine for now but when you use liners, it will show an error as it does not allow to use of more than 50 words in a single line. So it would be better to be linter friendly from now 😳